packages/test/test-end-to-end-tests/src/test/gc/gcDatastoreAliased.spec.ts

```diff
@@ -110,8 +108,9 @@ describeNoCompat("GC Data Store Aliased", (getTestObjectProvider) => {
         // Should be able to retrieve root datastore from remote
         const containerRuntime2 = mainDataStore2.containerRuntime;
         assert.doesNotThrow(async () => containerRuntime2.getRootDataStore(alias), "Aliased datastore should be root!");
```

Comment:
This should also be done before setting the alias and it should throw.

Reply:
We already have a test for this scenario in `rootDatastores.spec.ts`